### PR TITLE
[Affiliate] Add FAQ

### LINKED ISSUE #1683: [Affiliate] FAQ
# Summary

Add FAQ entry regarding affiliate program/profile page

First step is to define the text based on some/all topics below.
Second step is to add this to the UI

## Topics to cover
* What is the Profile page
* In which networks it's available
* What are the rewards for sharing the referral link
* What are the rewards for following the referral link
* Who can share
* Who can follow
* What is needed to qualify

Doc with proposed content https://docs.google.com/document/d/1BC66i33FT3eAK3o3B01uM7xDha9fJ1pvNN5TuBBzwCA/edit?usp=sharing

### LINKED ISSUE #1776: [Affiliate] Add 'later' to several error messages
Related to #1575, #1675

It would be nice to add  'later' to the phrase 'Please try again.' into several error messages. 
In particular: 

 - Affiliate program error: There was an error validating existing trades. Please try again **later**.

 - Affiliate program error: There was an error while uploading the referral document to IPFS. Please try again **later**.

### PR DESCRIPTION
**Pull request by alongoni**
# Summary

Fixes #1683
Also Closed #1776 
Affiliate program information added to FAQ. [Source information.](https://docs.google.com/document/d/1BC66i33FT3eAK3o3B01uM7xDha9fJ1pvNN5TuBBzwCA/edit#heading=h.ca8szqjs3xd1)

image
  # To Test

1. Open the [page `faq`](https://pr1691--gpswapui.review.gnosisdev.com/#/faq)

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @alongoni , I have found that we have one question sounds a bit different than in the document
name

In addition, one question is missing there, but I'm not really sure whether it should present there...
ьшыыштп

**Review by anxolin — APPROVED** [review]
It looks great! 

As a nitipick, re-reading it, these 2 last questions feel a bit too long. Maybe theres ways to simplify
image

**Comment by alongoni** [comment]
> Hey @alongoni , I have found that we have one question sounds a bit different than in the document name
> 
I'll fix that.
> In addition, one question is missing there, but I'm not really sure whether it should present there... ьшыыштп

This text is part of the [PR#1721](https://github.com/gnosis/cowswap/pull/1721)

**Comment by avsavsavs** [comment]
@anxolin maybe one way to simplify the last two questions can simply just be merging them and getting rid of the question title for the last one.

File: src/custom/pages/Faq/index.tsx [hunk 1]
```diff
@@ -745,6 +745,121 @@ export default function Faq() {
...
+          <h3 id="what-is-the-profile-page">What is the Profile page?</h3>
+
+          <p>
+            It’s a page where you can see your number of trades and volume that you have done with the wallet you have
+            connected with.
+          </p>
+
+          <p>
+            Additionally, you will be able to get your own referral/affiliate link, so you can share it with others, and
+            if you do so, you will also be able to see the # of trades and volume that your referral link gets credited
```
Comment (alfetopito) [inline src/custom/pages/Faq/index.tsx:762]:
```suggestion
            if you do so, you will also be able to see the number of trades and volume that your referral link gets credited
```

File: src/custom/pages/Faq/index.tsx [hunk 2]
```diff
@@ -745,6 +745,121 @@ export default function Faq() {
...
+
+          <p>At this time, the affiliate program only works for Ethereum Mainnet.</p>
+
+          <h3 id="are-there-any-rewards-for-sharing-the-referral-link">
+            Are there any rewards for sharing the referral link?
+          </h3>
+
+          <p>
+            The affiliate program will initially run as a trial, and therefore there are currently no announced rewards,
+            the tracing, however, has already started.
```
Comment (alfetopito) [inline src/custom/pages/Faq/index.tsx:776]:
```suggestion
            the tracking however, has already started.
```

File: src/custom/pages/Faq/index.tsx [hunk 3]
```diff
@@ -745,6 +745,121 @@ export default function Faq() {
...
+              Your friend was trading tokens that don’t have a price feed available (see FAQ entry{' '}
+              <LinkScrollable href={'#cwhy-is-my-total-trade-referral-trade-volume-smaller-than-the-real-volume?'}>
+                Why is the volume smaller than the real volume?
+              </LinkScrollable>
+              ).
+            </li>
+            <li>The data has not yet propagated from the chain to our backend.</li>
+            <li>
+              Your friend had already traded with that account, which does not count towards referral volume (see FAQ
+              entry Who can follow the referral link?).
```
Comment (alfetopito) [inline src/custom/pages/Faq/index.tsx:844]: Missing link to FAQ entry

**Review by alfetopito — CHANGES_REQUESTED** [review]

**Review by alfetopito — COMMENTED** [review]
Changes look great!

One thing I just noticed, sorry about that

Screenshot_20211110_091720

Can you move the bottom of the previous sections to the new one so that text shows last in the page?

**Review by alfetopito — APPROVED** [review]

**Comment by elena-zh** [comment]
LGTM!

**Pull request merged**